src/it/scala/org/apache/spark/sql/SFDataFrameAggregateSuite.scala

```diff
@@ -0,0 +1,89 @@
+package org.apache.spark.sql
+
+import org.apache.spark.sql.snowflake.{SFQueryTest, SFTestData, SFTestSessionBase}
+
+class SFDataFrameAggregateSuite
+    extends DataFrameAggregateSuite
+    with SFTestSessionBase
+    with SFQueryTest
+    with SFTestData {
+
+  override def spark: SparkSession = getSnowflakeSession()
+
+  override protected def blackList: Seq[String] =
+    Seq(
+      // contains RDD, replaced by TS - groupBy
```

Comment:
I believe you should modify these black lists

Reply:
This will be addressed in a later PR.